hotel-management/src/components/RoomList/index.tsx

```diff
@@ -8,15 +12,24 @@ type RoomListProps = {
   rooms: Room[];
 };
 
-export const RoomList = ({ rooms }: RoomListProps) => {
+export const RoomList = ({ rooms: initialRooms }: RoomListProps) => {
+  const [rooms, setRooms] = useState(initialRooms);
```

Comment:
This seems wrong, what happen if props rooms is changed and the roomlist is not change